react/features/video-menu/components/native/VolumeSlider.tsx

```diff
@@ -123,10 +134,22 @@ class VolumeSlider extends PureComponent<IProps, IState> {
      * @private
      * @returns {void}
      */
-    _onVolumeChange(volumeLevel: number) {
-        const { dispatch, participantID } = this.props;
+    _onVolumeChange(volumeLevel: any) {
+        const { _track, dispatch, participantID } = this.props;
+        const audioTrack = _track?.jitsiTrack.track;
+
+        let newVolumeLevel;
 
-        dispatch(setVolume(participantID, volumeLevel));
+        if (volumeLevel <= 10) {
+            newVolumeLevel = volumeLevel / 10;
+        } else {
+            newVolumeLevel = volumeLevel;
+        }
```

Comment:
This doesn't look right. If the new volume is 0-10 we do want it divided by 10. If it's 11-20 we need to put it in the (1-10) range so we need to substract 10.

Now, that would mean that 10 and 11 are the same value, which is arguably wrong, so maybe we should have the scale go from 0 to 19, so 20 elements total.

0-10: value / 10
11-19: value -9